### DISCUSSION TITLE
Support for Business Login for Instagram

### DISCUSSION THREAD

**Discussion by kerryjj** [thebergamo/react-native-fbsdk-next#460] [Ideas]
Hi there,

Meta seems to have introduced new feature recently called Business Login for Instagram.

https://developers.facebook.com/docs/instagram/business-login-for-instagram

Are there any plans to support this in react-native-fbsdk-next?

**Comment by yogevlahyani** [comment]
Any updates?
Trying to pass extras={\"setup\":{\"channel\":\"IG_API_ONBOARDING\"}} to the url for login button
